eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php

```diff
@@ -107,13 +147,12 @@ public function index(FullTextData $fullTextData)
             } else {
                 $integerValue = 0;
             }
-            // Split text on whitespace
-            $wordArray = explode(' ', $fullTextValue->value);
+            // Split transformed text on whitespace
+            $wordArray = explode(' ', $this->transformationProcessor->transform($fullTextValue->value, $this->transformationProcessorCommands));
```

Comment:
It seems we need to extract a service to inject in both places. But this is fine for now, you can refactor that later.